src/lib/analytics/analytics.js

```diff
@@ -101,42 +108,74 @@ const identifyWith = (email, identifier = null) => {
     Rollbar.configure({
       payload: {
         person: {
-          id: identifier,
           email,
         },
       },
     })
   }
 
   if (isAmplitudeEnabled) {
-    if (identifier) {
-      Amplitude.setUserId(identifier)
-    }
+    Amplitude.setUserProperties({ email })
+  }
 
-    if (email) {
-      Amplitude.setUserProperties({ email })
+  if (FS) {
+    FS.setUserVars({
+      email,
+    })
+  }
+
+  if (isSentryEnabled) {
+    Sentry.configureScope(scope => {
+      const { _user } = scope
+
+      scope.setUser({
+        ...(_user || {}),
+        email,
+      })
+    })
+  }
+
+  if (Mautic) {
+    Mautic.userId = email
+  }
+}
+
+const identifyWith = (email, identifier = null) => {
```

Comment:
please also set the email in this method in all services

Reply:
it sets email by calling setUserEmail(email):
https://github.com/GoodDollar/GoodDAPP/blob/66ced8a577c8b9939311a02c46c1aa98ca6fa4c6/src/lib/analytics/analytics.js#L178